setup.cfg

```diff
@@ -13,3 +13,21 @@ line_length=88
 [flake8]
 max-line-length = 88
 extend-ignore = E203
+
+[mypy-django.*]
+ignore_missing_imports = True
+
+[mypy-django_countries.*]
+ignore_missing_imports = True
+
+[mypy-django_filters.*]
+ignore_missing_imports = True
+
+[mypy-mptt.*]
+ignore_missing_imports = True
+
+[mypy-rest_framework.*]
+ignore_missing_imports = True
+
+[mypy-rest_framework_gis.*]
+ignore_missing_imports = True
```

Comment:
why are these necessary?

Reply:
mypy raises import errors for [third party packages that it cannot find](https://mypy.readthedocs.io/en/stable/existing_code.html#start-small), had to silence them so that they do not cause the checks to fail.

Reply:
Hmmm, I am hesitant about this - because I am strongly against relaxing linting rules.  Some questions:

- why cant mypy find these packages (I ask this because I have used it elsewhere and not gotten into this problem)?

Reply:
> Hmmm, I am hesitant about this - because I am strongly against relaxing linting rules. Some questions:
> 
>     * why cant mypy find these packages (I ask this because I have used it elsewhere and not gotten into this problem)?

Seems like this was just a blunder on my side. Was running it outside of my virtualenv.

After running it in the virtualenv it doesn't seem to raise any errors without the config. I'll make the change.